### PR TITLE
Revert falcon exception

### PR DESCRIPTION
**Pull request by LysandreJik**
Reverts the Falcon exceptions

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
Niiice

**Review by Rocketknight1 — APPROVED** [review]
:pray:

**Comment by LysandreJik** [comment]
We'll merge this once your PRs are merged @Rocketknight1 !

**Pull request merged**
